tests/test_SampleData.py

```diff
@@ -1,17 +1,49 @@
 """Test MolecularDynamicsData"""
 
-from SimExLite.SampleData import MolecularDynamicsData
 import h5py
+import numpy as np
+import pytest
+from SimExLite.SampleData import SampleData, ASEFormat
 
 
-def test_XMDYNData():
-    MDD = MolecularDynamicsData("./testFiles/xmdyn.h5")
-    assert MDD.file_format == "XMDYN"
+@pytest.fixture(scope="session")
+def tmp_data():
+    data_dict = {}
+    # 2 atoms
+    positions = np.empty((2, 3))
+    positions[0] = np.array([8, 18, -3.54])
+    positions[1] = np.array([9, 18.2, -3.1])
+    data_dict["positions"] = positions
+    data_dict["atomic_numbers"] = np.array([8, 6])
+    return data_dict
 
 
-def test_UnknownData(tmp_path):
-    tmp_h5 = str(tmp_path / 'tmp.h5')
-    with h5py.File(tmp_h5, 'w') as h5:
-        h5.create_group('test_g')
-    MDD = MolecularDynamicsData(tmp_h5)
-    assert MDD.file_format == "UNKNOWN"
```

Comment:
is it best pracice to use vanilla `assert` in pytest test modules?

Reply:
Yes, according to its documentation: https://docs.pytest.org/en/6.2.x/getting-started.html?highlight=assert
And it works well.